cf/src/main/scala/com/timeout/scalacloudformation/Template.scala

```diff
@@ -0,0 +1,12 @@
+package com.timeout.scalacloudformation
+
+import io.circe.Json
+
+
+case class Template(Description: Option[String] = None,
+                    Metadata: Option[Json] = None,
+                    Mappings: Option[Json] = None,
```

Comment:
I implemented this as a simple `Mappings: Map[String, Map[String, String]]` since it's always a two-levels map of strings, without expression...
http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/mappings-section-structure.html